advocate_europe/assets/scss/components/_blocks.scss

```diff
@@ -125,9 +125,26 @@
     }
 
     .blockblog-image {
+        background-size: cover;
```

Comment:
Since this is roughly used three times, you might consider moving this into an extra class or mixin for those classes in an extra scss module.